src/beryl/router.cr

```diff
@@ -61,31 +66,35 @@ module Beryl
     # important when defining them. Is recommended you define the shorter
```

Comment:
Now that we use a Radix Tree, the order on which the routes are defined is irrelevant.

Now a tree is built for each request type and the route is looked up inside the tree by using the request's `path`.